packages/openapi-generator/src/document-converter.ts

```diff
@@ -75,47 +75,32 @@ export async function convertDocToOpenApiV3(
  * @param openApiDocument OpenAPI JSON document.
  * @returns The modified document.
  */
-export function convertDocWithDefaultTag(
+export function convertDocWithApiNameTag(
   openApiDocument: OpenAPIV3.Document
 ): OpenAPIV3.Document {
-  const tag = 'default';
+  const defaultTag = 'default';
 
-  executeForAllOperationObjects(openApiDocument, operation => {
-    operation.tags = addGlobalTagToOperationWhenNoTagsAreUsed(
+  executeForAllOperationObjects(openApiDocument, (operation,path, method, extensionApiName) => {
```

Comment:
With four arguments I think it makes sense to transform this to an object - that way you won't have to pass unused parameters as in this case.